internal/controlplane/http.go

```diff
@@ -46,4 +46,7 @@ func (srv *Server) addHTTPMiddleware() {
 	root.Path("/debug/pprof/symbol").HandlerFunc(pprof.Symbol)
 	root.Path("/debug/pprof/trace").HandlerFunc(pprof.Trace)
 	root.PathPrefix("/debug/pprof/").HandlerFunc(pprof.Index)
+
+	// metrics
+	root.Handle("/metrics", srv.metricsMgr)
```

Comment:
I'm not sure we want to also expose all the above special handlers for a metrics endpoint.

Reply:
Do you want me to create a separate control plane HTTP server for metrics?